Cheatsheet.md

```diff
@@ -0,0 +1,554 @@
+# Cheatsheet
+### Commonly used [array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays) functions

```

Comment:
Rephrase it to `Commonly used functions on Array` and make it header 2 instead of header 3.